src/pages/gen2/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -289,7 +289,8 @@ If you are using the [Authenticator component](https://ui.docs.amplify.aws/react
 Enabling a signing in of a user with a social provider can be done as shown in the following code snippet.
 
 ```ts title="src/my-client-side-js.js"
-import { signInWithRedirect } from 'aws-amplify/auth';
+import 'aws-amplify/auth/enable-oauth-listener'; // required for multi-page applications
```

Comment:
Shouldn't we add the whole section explaining why this is required? 
https://docs.amplify.aws/react/build-a-backend/auth/add-social-provider/#required-for-multi-page-applications-complete-social-sign-in-after-redirect

Reply:
makes sense to me! added